spec/shr_adverse.txt

```diff
@@ -0,0 +1,88 @@
+Grammar:		DataElement 5.0
+Namespace:		shr.core
+
+CodeSystem:		MTH = http://ncimeta.nci.nih.gov 
+
+
+// Adverse drug reactions are a subset of adverse events, and are specifically associated with an exposure to a substance
+
+
+Element:		AdverseEvent
+Based on:		ClinicalStatement
+Description:	"A record of an untoward or unexpected event in a patient under treatment, and which does not necessarily have to have a causal relationship with that treatment. Adverse events may capture either a workflow breach that may or may not result in harm, but must be documented for quality improvement purposes, or an adverse finding whether or not it can be traceable to an actual workflow breach."
+0..1			Identifier
+				Recorder value is type PatientOrPractitionerOrRelatedPerson
+1..1			Patient
+1..1			OccurrenceTime
+0..1			Type from MedDRAVS
```

Comment:
Adverse event is based on CTCAE 5.0, then we can replace these with NCI thesaurus rather than MedDRA.  Reason is that, like CPT, MedDRA is proprietary, and might require licensing depending on whether the implementer is non-profit or commercial.

Reply:
That's a good idea. Can you suggest the right value set?

Reply:
@markkramerus - An extensional value set can easily be curated since the CTCAE concepts are hierarchical within the NCI thesaurus. In this link they would be the distinct concepts 2 child levels below "CTCAE by Body System" (NCIt#143163): https://ncit-stage.nci.nih.gov/ncitbrowser/ajax?action=search_hierarchy&ontology_node_id=C143163&ontology_node_ns=ncit&ontology_display_name=NCI_Thesaurus&version=19.02d

One could potentially create an intensional value set through a REST-based service in the BioOntology portal, but it requires one setup a free account to receive an API key that an application will need to pass in the API query.  

Since CTCAE 5.0 does not change much, if at all, I suggest we simply create the extensional (static) value set.